src/cook-web/src/app/c/[[...id]]/Components/ChatUi/ChatUi.module.css

```
@@ -78,25 +88,58 @@
 }
+.ChatUi .footer.hideMobileFooter {
+  display: none !important;
+}
 .ChatUi.mobile {
   width: 100%;
-  padding-bottom: calc(40px + env(safe-area-inset-bottom, 0px));
+  padding-bottom: 0;
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  height: 100dvh;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

height is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)